src/components/dialog/Dialog.jsx

```diff
@@ -118,17 +113,26 @@ function BaseDialog({
     [open, lastTransitionName, onEntryTransitionEnd, onExitTransitionEnd]
   );
 
+  const handleKeyUp = React.useCallback(
```

Comment:
When we manage to fix focus trap (if we confirm it's an issue introduced by this PR), we probably wan't to remove `useEscapeKey` hook entirely.

Reply:
oops I didn't notice it's already removed. Ignore